modules/saml/lib/SP/LogoutStore.php

```diff
@@ -14,12 +14,19 @@ class sspmod_saml_SP_LogoutStore {
 	 */
 	private static function createLogoutTable(\SimpleSAML\Store\SQL $store) {
 
-		if ($store->getTableVersion('saml_LogoutStore') === 1) {
+		if ($store->getTableVersion('saml_LogoutStore') === 2) {
+			return;
+		}
+		elseif ($store->getTableVersion('saml_LogoutStore') === 1) {
+			/* TableVersion 2 increased the column size to 255 which is the maximum length of a FQDN. */
+			$query = 'ALTER TABLE' . $store->prefix . '_saml_LogoutStore MODIFY _authSource VARCHAR(255) NOT NULL';
+			$store->pdo->exec($query);
+			$store->setTableVersion('saml_LogoutStore', 2);
```

Comment:
Should we check the result of the `ALTER TABLE` statement before bumping the table version? If something goes wrong (i.e. this particular syntax is unsupported by the database backend in use), then we should keep the version to `1` (and probably log an error).